Content.Server/Atmos/EntitySystems/AtmosphereSystem.HighPressureDelta.cs

```diff
@@ -181,12 +184,37 @@ private void ConsiderPressureDifference(GridAtmosphereComponent gridAtmosphere,
             tile.PressureDirection = differenceDirection;
         }
 
+        //INFO:The EE version of this function drops pressureResistanceProbDelta, since it's not needed. If you are for whatever reason calling this function
```

Comment:
```suggestion
        //INFO The EE version of this function drops pressureResistanceProbDelta, since it's not needed. If you are for whatever reason calling this function
```